examples/metalk8s-solution-example/ui/src/containers/Layout.js

```diff
@@ -49,20 +49,6 @@ const Layout = props => {
 
   const applications = [];
```

Comment:
you should update the core-ui in package.json of metalk8s-solution-example as well